src/XfdfFile.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace mikehaertl\pdftk;
+
+use mikehaertl\tmp\File;
+
+/**
+ * XfdfFile
+ *
+ * This class represents a temporary XFDF file that can be used to fill a PDF form
+ * with valid unicode characters.
+ *
+ * @author Tomas Holy <holy@interconnect.cz>
+ * @version 0.2.2
+ * @license http://www.opensource.org/licenses/MIT
+ */
+class XfdfFile extends File
+{
+    // XFDF file header
+    const XFDF_HEADER = <<<FDF
+<?xml version="1.0" encoding="UTF-8"?>
+<xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve">
+<fields>
+FDF;
+
+    // XFDF file footer
+    const XFDF_FOOTER = <<<FDF
+</fields>
+</xfdf>
+FDF;
+
+    /**
+     * Constructor
+     *
+     * @param array $data the form data as name => value
+     * @param string|null $suffix the optional suffix for the tmp file
+     * @param string|null $suffix the optional prefix for the tmp file. If null 'php_tmpfile_' is used.
+     * @param string|null $directory directory where the file should be created. Autodetected if not provided.
+     * @param string|null $encoding of the data. Default is 'UTF-8'.
```

Comment:
The `$encoding` parameter is not used. So either we should remove it, or use the same code as in `FdfFile` to convert from `$encoding` to `UTF-8`.